custom/icds_reports/views.py

```diff
@@ -548,11 +548,25 @@ def get(self, request, *args, **kwargs):
             )
         elif step == 'beneficiary':
             if 'awc_id' in config:
+                start = request.GET.get('start', 0)
+                length = request.GET.get('length', 10)
+                draw = request.GET.get('draw', 0)
+
+                order_by_number_column = request.GET.get('order[0][column]')
```

Comment:
Does this work? I've never seen something like this before and I don't see an example in django's docs

Reply:
@emord Yes this works, This strange key is added by angular-datatable automatically.

Reply:
Thanks. Didn't realize this was from angular